.github/workflows/Test.yaml

```diff
@@ -13,37 +13,33 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-latest"]
-        python-version: ["3.9", "3.10"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        ambertools: ["true", "false"]
```

Comment:
I don't think there's a huge value in doubling this to 8 builds.

I'd just throw in a step to install ambertools if python != 3.12